microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,78 @@
+---
+description: Feature differences between Microsoft Edge and WebView2
+title: Feature differences between Microsoft Edge and WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/08/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, WebView2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Feature differences between Microsoft Edge and WebView2  
+
+WebView2 is based on the new Microsoft Edge browser.  You \(a WebView2 developer\) have the opportunity to extend features from the Microsoft Edge to WebView2-based apps.  WebView2 includes various use cases and isn't limited to only being a browser.  Some of the Microsoft Edge use cases for browser features should work within the context of WebView2.  
```

Comment:
If this is part of Microsoft docs guidance then go for it. Otherwise it looked unusual and unnecessary to me as I don't imagine we have more than one audience for this doc (I can be wrong about that too)

Reply:
The expanded audience description was retained from a previous version of the document.  The use of `you` is preferred.